diagnostic.md

```diff
@@ -76,7 +76,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+_2 loafs of bread, 2 contaniers of milk, 2 sticks of butter_
```

Comment:
Remember that the index of an array starts at 0. So groceryList[0] would be,
'a loaf of bread'; groceryList[1] would be 'a container of milk' and grocery[2] would be 'a stick of butter', which is the answer. The groceryList[_value_] is which object in the array it's referencing.